StripeConnect/StripeConnect/Source/Components/ComponentType.swift

```diff
@@ -8,11 +8,32 @@
 import Foundation
 
 /// The name of the embedded component tag in JS ([docs](https://docs.stripe.com/connect/supported-embedded-components))
-enum ComponentType: String, Encodable {
+enum ComponentType: Encodable {
     /// Displays the balance summary, the payout schedule, and a list of payouts for the connected account
     case payouts
     /// The onboarding flow for the account.
-    case onboarding = "account-onboarding"
+    case onboarding(AccountOnboardingViewController.Props)
     /// Show details of a given payment and allow users to manage disputes and perform refunds.
-    case paymentDetails = "payment-details"
+    case paymentDetails
```

Comment:
Adding init params to the `ComponentType` so that they can be passed on init to `ComponentWebView`. 

👀 Should this get renamed to `Component` maybe?